src/eme.js

```diff
@@ -59,7 +59,18 @@ export const getSupportedKeySystem = (keySystems) => {
   let promise;
 
   Object.keys(keySystems).forEach((keySystem) => {
-    const supportedConfigurations = getSupportedConfigurations(keySystems[keySystem]);
+    const options = Object.assign({}, keySystems[keySystem]);
+
+    if (keySystem === 'com.widevine.alpha') {
+      if (!Object.hasOwnProperty('audioRobustness')) {
+        options.audioRobustness = 'SW_SECURE_CRYPTO';
```

Comment:
Since the default is an empty string, will this potentially cause any unexpected behavior (for instance, an environment where only hardware crypto is used)?